hack/test/e2e.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+set -eou pipefail
+
+export TMP="$(mktemp -d)"
+export OSCTL="${PWD}/build/osctl-linux-amd64"
+export TALOSCONFIG="${TMP}/talosconfig"
+export KUBECONFIG="${TMP}/kubeconfig"
+
+##ClusterAPI Provider Talos (CAPT)
```

Comment:
Total nit: 1 `#` plus  a space between the text:
```
# ClusterAPI Provider Talos (CAPT)
```

Reply:
Fixed